apps/full-site-editing/full-site-editing-plugin/block-patterns/class-block-patterns.php

```diff
@@ -82,26 +82,51 @@ public function get_patterns() {
 			return $patterns;
 		}
 
-		$pattern = readdir( $directory_handle );
+		// Get all the pattern files.
+		$pattern       = readdir( $directory_handle );
+		$pattern_files = array();
 		while ( false !== $pattern ) {
-			if ( substr( $pattern, -5 ) === '.json' ) {
+			// Only allow files ending in .json or .php.
+			if ( substr( $pattern, -5 ) === '.json' || substr( $pattern, -4 ) === '.php' ) {
+				$pattern_files[] = $pattern;
+			}
+
+			$pattern = readdir( $directory_handle );
+		}
+
+		closedir( $directory_handle );
+
+		// Patterns to remove / disable.
+		$disabled_patterns = array();
```

Comment:
Do we really want to introduce a functionality like, rather than just removing the pattern files that should be removed?

Reply:
I'm fine removing this. Would be easy to add back in if necessary. cc @ianstewart

Reply:
Same as Jerry. Agreed.

Reply:
Removed.